tests/logictest/suites/duckdb/sql/aggregate/having/test_having.test

```diff
@@ -54,6 +54,7 @@ SELECT test.b, SUM(a) FROM test GROUP BY test.b HAVING SUM(a)=(SELECT SUM(a) FRO
 
 ----
 21 12
```

Comment:
The original test result was wrong... 😅 


<img width="1309" alt="image" src="https://user-images.githubusercontent.com/41979257/196328333-843a2018-4a99-4b98-9909-daf2ad2c6bed.png">